### PR TITLE
Bk 721 create suggestions request

### PR DESCRIPTION
**Pull request by mrroot5**
Upgrade package version and added suggest request for search API v6

### CODE REVIEW

**Comment by mrroot5** [comment]
If I do something wrong, please told me to improved my "PR flow"

File: pydoof/search_api/search.py [hunk 1]
```diff
@@ -81,54 +81,30 @@ def query(hashid, query, filter_=None, exclude=None, index_name=None,
     )
 
 
-def suggest(hashid, query, filter_=None, exclude=None, sort=None, page=None,
-            rpp=None, transformer=None, no_stats=None, **opts):
+def suggest(hashid, query, indices=None, stats=None, session_id=None, **opts):
```
Comment (sonic182) [inline pydoof/search_api/search.py:84]:
```suggestion
from typing import List
# ...
def suggest(hashid: str, query: str, indices: List[str]=None, stats: bool=None, session_id=None, **opts):
```

We can start using python typing in this package also

**Review by sonic182 — APPROVED** [review]

**Review by mjosecuchillo — APPROVED** [review]

File: pydoof/search_api/search.py [hunk 2]
```diff
@@ -81,54 +81,32 @@ def query(hashid, query, filter_=None, exclude=None, index_name=None,
     )
 
 
-def suggest(hashid, query, filter_=None, exclude=None, sort=None, page=None,
-            rpp=None, transformer=None, no_stats=None, **opts):
+from typing import List
```
Comment (sonic182) [inline pydoof/search_api/search.py:84]: But... put this at top, don't commit the suggestion as it is please :) at least the one I did before (that was not a simpler-one)

**Review by sonic182 — CHANGES_REQUESTED** [review]

File: pydoof/search_api/search.py (hunk 2 above)
Comment (mrroot5) [reply pydoof/search_api/search.py]: Ok, I do this on next commit. Thanks!

**Review by mdogo — APPROVED** [review]

**Review by sonic182 — APPROVED** [review]

**Review by mjosecuchillo — APPROVED** [review]

**Pull request merged**